Makefile

```diff
@@ -12,6 +12,15 @@ test: image
 	  --volume "$(PWD)":/app \
 	  codeclimate/minidoc bundle exec rspec $(RSPEC_ARGS)
 
+# Route named service hosts to the docker bridge
+citest: DOCKER_BRIDGE_IP=$(shell ip addr show dev docker0 | sed '/^.*inet \(.*\)\/.*$$/!d; s//\1/')
+citest:
+	docker run \
+	  --add-host mongodb:$(DOCKER_BRIDGE_IP) \
+	  --env MONGODB_URI="$(MONGODB_URI)" \
+	  --volume $(PWD):/app \
+	  codeclimate/minidoc bundle exec rspec && ./cc-test-reporter after-build -p "/app"
```

Comment:
one small thing -- we could use `--prefix` over `-p` in the interest of being more explicit